index.js

```diff
@@ -26,14 +26,16 @@ async function resolveVersion(version, mirror) {
   let mirror = core.getInput("node-mirror") || "https://nodejs.org/dist/";
   let version = await resolveVersion(core.getInput("node-version"), mirror);
   if (process.platform == "win32") {
-    runScript("powershell", ".\\install.ps1", version, mirror);
+    let arch = core.getInput("node-arch") || "";
```

Comment:
should this throw if provided on a different platform?

Reply:
Good question. In this particular case, _node-arch_ is only being processed on Windows systems and I marked the input field as "Windows only" in the [README file](https://github.com/dcodeIO/setup-node-nvm/blob/4aa073e40593e1d35c42126748a60c6710aa72d7/README.md#node-arch-windows-only) in an attempt to avoid any confusion between nvm and nvm-windows.

As an option, we can move this out of the platform check and indicate that _node-arch_ has no effect on *nix systems when provided, but IMHO, throwing doesn't look like a good idea to me. But of course, I'm happy to hear your thoughts on this.

Reply:
I can see the benefit of silently ignoring it, when the user's intention is "only respect this on windows".

However, if the user's intention is "force this architecture", they might end up running non-windows tests on a different architecture than intended, which could mask a lot of problems.

Since it's impossible to know the user's intention, it seems to me like the safer approach is to throw rather than silently ignore the invalid option.

Reply:
Sure, I see your point, however, can you elaborate a bit more on:

> if the user's intention is "force this architecture", they might end up running non-windows tests on a different architecture than intended, which could mask a lot of problems.

I'm not sure I follow you. How the users might end up doing such thing if the option is ignored on non-windows platforms?
I just want to understand this better, that's all. 

Oh, and throwing on non-windows platforms should be as easy as adding a _core.setFailed_ when _node-arch_ has been provided, @dcodeIO would you be okay with it?

Reply:
If a user's intention is to ensure their project works on 32 bit and 64 bit machines, and in fact their non-windows tests are only running on 64 bit machines, then their 32-bit users on non-windows machines aren't being covered - the testing is a false positive.